Automattic/wp-calypso
# Remove the client/document/desktop module

This module is no longer used since #49614 removed the call to `renderJsx( 'desktop', context )`.

**How to test:**
Inspect all `renderJsx` calls in server code and verify that none of them renders the `desktop` view.
